include/ghostty.h

```diff
@@ -762,6 +762,7 @@ typedef enum {
   GHOSTTY_ACTION_RESIZE_SPLIT,
   GHOSTTY_ACTION_EQUALIZE_SPLITS,
   GHOSTTY_ACTION_TOGGLE_SPLIT_ZOOM,
+  GHOSTTY_ACTION_TOGGLE_READONLY,
```

Comment:
Same feedback here as your other PR, we don't need an apprt action for this.

We should have an apprt action `GHOSTTY_ACTION_READONLY` or something that sends a boolean with it that notes the current state of the surface. 

This way GUIs can eventually (not this PR) show that the terminal is in a read-only state.